src/components/footnote/footnote.stories.mdx

```diff
@@ -0,0 +1,48 @@
+import { Story, Canvas, Meta } from '@storybook/addon-docs/blocks';
+import template from './demo/demo.twig';
+// eslint-disable-next-line @cloudfour/import/no-webpack-loader-syntax
+import demoSource from '!!raw-loader!./demo/demo.twig';
+
+<Meta
+  title="Components/Footnote/Footnote"
+  argTypes={{
+    count: { type: { name: 'number' } },
+    id: { type: { name: 'number' } },
+    content: { type: { name: 'string' } },
+  }}
+/>
+
+# Footnote
+
+Used for displaying the content of a footnote.
+
+Renders as a list item (`<li>`). Expected to be wrapped in a [Footnote Group](/docs/components-footnote-footnote-group--basic) and paired with a [Footnote Link](/docs/components-footnote-footnote-link--basic).
+
+## Basic usage
+
+<Canvas>
+  <Story
+    name="Basic"
+    args={{
+      count: 1,
+    }}
+    parameters={{
+      docs: {
+        source: {
+          code: demoSource,
+        },
+      },
+    }}
+  >
+    {(args) => template(args)}
+  </Story>
+</Canvas>
+
+## Template Properties
+
+- `count` (number)
```

Comment:
@Paul-Hebert Can you add a description here too, thanks! :)

Reply:
Done! https://github.com/cloudfour/cloudfour.com-patterns/pull/1518/commits/c2a2cd9738e947e7f6930d4b5650777274ee6b11